modules/admin_manual/pages/configuration/server/logging/logging_configuration.adoc

```diff
@@ -73,9 +76,7 @@ For the old syslog message format use:
 
 === Conditional Logging Level Increase
 
-You can configure the logging level to automatically increase to `debug`
-when the first condition inside a condition block is met. All conditions
-are optional !
+xref:configuration/server/config_sample_php_parameters.adoc#define-log-conditions[Log conditions] for log level increase based on conditions can be set. This will incfrease the logging level to automatically to `debug` when the first condition inside a condition block is met. All conditions are optional !
```

Comment:
```suggestion
xref:configuration/server/config_sample_php_parameters.adoc#define-log-conditions[Log conditions] for log level increase based on conditions can be set. This will increase the logging level automatically to `debug` when the first condition inside a condition block is met. All conditions are optional !
```